features/remote-playback.yml

```
@@ -0,0 +1,18 @@
+name: Remote playback
+description: The Remote Playback API allows webpages to initiate and control playback of media on connected remote devices, such as smart TVs with AirPlay or Chromecast capabilities.
```

I think we want to avoid "allow" in descriptions.

```suggestion
description: The Remote Playback API initiates and controls playback of media on connected remote devices, such as smart TVs with AirPlay or Chromecast capabilities.
```